src/gui/properties/propertieswidget.cpp

```diff
@@ -442,11 +442,11 @@ void PropertiesWidget::loadDynamicData()
 
             m_ui->labelDlSpeedVal->setText(tr("%1 (%2 avg.)", "%1 and %2 are speed rates, e.g. 200KiB/s (100KiB/s avg.)")
                 .arg(Utils::Misc::friendlyUnit(m_torrent->downloadPayloadRate(), true)
-                    , Utils::Misc::friendlyUnit(m_torrent->totalDownload() / (1 + m_torrent->activeTime() - m_torrent->finishedTime()), true)));
+                    , Utils::Misc::friendlyUnit(m_torrent->totalDownload() / qMax((m_torrent->activeTime() - m_torrent->finishedTime()), 1), true)));
```

Comment:
It seems like this formula is wrong in case of incorrect time value (and below too).

Reply:
what do you mean/suggest?

Reply:
On another note: IIRC in other places we use std::max(), so why qMax() here?

Reply:
a bit shorter to type and QtGlobal header is included almost universally, I don't recall we have a preference for one.

Reply:
OK

>this formula is wrong in case of **incorrect time** value

He probably means that we shouldn't fudge the numbers if we don't have **valid** values.
IMO, in this case it is better to print "Uknown avg." like how it's done when friendlyUnit() is passed a negative number.

Reply:
>He probably means that we shouldn't fudge the numbers if we don't have valid values.

Yes.

Reply:
OK, updated.